themes/theme-gmd/widgets/Liveshopping/components/Discount/index.jsx

```diff
@@ -1,12 +1,12 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
-import DiscountBadge from '@shopgate/pwa-ui-shared/DiscountBadge';
-import Portal from '@shopgate/pwa-common/components/Portal';
+import { Portal, DiscountBadge } from '@shopgate/engage/components';
 import {
   PRODUCT_ITEM_DISCOUNT,
   PRODUCT_ITEM_DISCOUNT_BEFORE,
   PRODUCT_ITEM_DISCOUNT_AFTER,
 } from '@shopgate/pwa-common-commerce/category/constants/Portals';
+import { i18n } from '@shopgate/engage/core';
```

Comment:
please change the import to `@shopgate/engage/core/helpers`, since we should avoid adding new imports from the index files.